### PR TITLE
Add classpath to CFR decompile

### PR DESCRIPTION
**Pull request by Arthurm1**
Adds the classpath to the args of CFR so the decompile is more accurate.

### CODE REVIEW

File: metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala [hunk 1]
```diff
@@ -525,16 +526,74 @@ final class FileDecoderProvider(
...
+
+    val args = extraClassPath :::
+      List(
+        // elideScala - must be lowercase - hide @@ScalaSignature and serialVersionUID for aesthetic reasons
+        "--elidescala",
+        "true",
+        // analyseAs - must be lowercase
+        "--analyseas",
+        "CLASS",
+        s"""$className"""
```
Comment (kpodsiad) [inline metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala:586]: Are triple quotes needed here?

**Review by kpodsiad — APPROVED** [review]
Looks good!

**Review by tgodzik — APPROVED** [review]

File: metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala (hunk 1 above)
Comment (Arthurm1) [reply metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala]: No - I'll remove

**Comment by Arthurm1** [comment]
CompletionLspSuite failure.  Got to be spurious so I'm merging

**Pull request merged**